### ISSUE TITLE
Compile Arakoon 1.9.3 on Ubuntu 16.04

### ISSUE THREAD

**Issue by maxux** [openvstorage/arakoon#71]
Hi,

Here you can find a script which makes Arakoon 1.9.3 compiles on a fresh (at least OpenvCloud) Ubuntu 16.04:
https://gist.github.com/maxux/5d8cd7c6ce819ae8aa03d4457a7088cb

Note: this gist contains a patch needed to compiles on "Core 113.33".
After that, Arakoon compiles fine on all last version of libs used.

All test runs (except etcd which was not installed).

Version installed:

```
bisect               1.3
camlbz2              0.6.0
camltc               0.9.3
core                 113.33.03
depext               0.9.1
lwt                  2.5.2
ocamlfind            1.6.2
ocplib-endian        0.8
quickcheck           1.0.2
redis                0.2.3
snappy               0.1.0
ssl                  0.5.2
uri                  1.9.2
```

**Comment by domsj** [comment]
Hi Maxim, we build arakoon using another version of core, see our docker file: https://github.com/openvstorage/arakoon/blob/1.9/docker/arakoon_debian/Dockerfile#L71 .
We'll move to ubuntu 16.04 sometime in the near future, at which time we may also bump the versions of our dependencies.
Btw which version of the ocaml compiler are you using? We currently use 4.02.3.

**Comment by maxux** [comment]
Looks like the same version for ocaml compiler: `ocamlbuild 4.02.3`
After some more deeper test, all tests seems not okay. I check some stuff.

**Comment by maxux** [comment]
Gist updated with a script used to build a `arakoon_1.9.3_amd64.deb` working on 16.04
Note that one test seems to fail, but I don't have more information about that.

**Comment by domsj** [comment]
resolved by #106
